docs/UserGuide.md

```diff
@@ -192,24 +193,24 @@ Examples:
 *  `delete appointment 7` Deletes the `appointment` at index `7`.
 *  `delete property 7` Deletes the `property` at index `7`.
 
-### Listing
+### 3.5 Listing
 
-#### Listing all properties and appointments : `list all`
+#### 3.5.1 Listing all properties and appointments : `list all`
 
 Shows a list of all properties and appointments in the app.
 
-#### Listing all properties : `list property`
+#### 3.5.2 Listing all properties : `list property`
 
 Shows a list of all properties in the app.
 
-#### Listing all appointments : `list appointment`
+#### 3.5.3 Listing all appointments : `list appointment`
 
 Shows a list of all appointments in the app.
 
 
-### Updating status
+### 3.6 Updating status
 
-#### Updating the status of a property : `update`
+#### 3.6.1 Updating the status of a property : `update`
 
 Updates the status of a property from Option to Purchase, to Sales and Purchase Agreement to Completion
```

Comment:
should we explain what does Option, Sales and Purchase Agreement, and Completion mean?

Reply:
@dvdweien to update this section